### DISCUSSION TITLE
[Scam Alert] Orbit Network

### DISCUSSION THREAD

**Discussion by saeedkousha** [Webamoozcom/warning-list#65] [Announcements]
<div dir="rtl">

**Project Name:** 
Orbit Network

**نام پروژه:** 
شبکه اوربیت 

**نماد پروژه:**
SLN, NYE, RKN

**اتفاقی که افتاده است:**

شرکت Orbit Network با ادعای فعالیت در حوزه ی بلاکچین و توسعه ی ارزهای دیجیتال در حال جمع آوری سرمایه با استفاده از رمزارزها است.

اختصاصی ایران هشدار- در ادامه معرفی طرح های پانزی جذب سرمایه به بررسی شرکتی می پردازیم که به تازگی فعالیت خود را در ایران آغاز کرده و سؤالات زیادی از سوی کاربران در رابطه با اعتبار آن مطرح شده است. این شرکت Orbit Network به آدرس وب سایت Orbit Network.net است.

Orbit Network با ادعای فعالیت در حوزه ی بلاکچین و توسعه ی ارزهای دیجیتال در حال جمع آوری سرمایه با استفاده از رمزارزها است. اشتراک گذاری سود حاصل از پروژه های بلاکچینی با اعضا اصلی ترین حربه به کارگرفته شده توسط شرکت به منظور جذب سرمایه گذار است. در سایت شرکت ادعا می شود با کیف پول های دیجیتالی از جمله تراست والت در حال همکاری است و خدمات خود را بر روی این کیف پول ها ارائه می دهد. همچنین شرکت مالک دو پلتفرم مهم به نام های orbitex و ioexchange است.

از مواردی که برخی برای اعتبار بخشی به فعالیت شرکت مطرح می نمایند ارزهای دیجیتال Sln، Nye coin و Rkn است که در سایت شرکت لیست شده است در حالی که پیش از این شرکت های کلاهبرداری زیادی وجود داشتند که ارزهای دیجیتال را در وب سایت خود لیست کرده بودند.

از ابهامات دیگری که در رابطه با صحت اعتبار شرکت وجود دارد عدم ثبت سند در رابطه با تیم توسعه و دفتر مرکزی آن است. وب سایت شرکت نیز بار علمی و مفهومی نداشته و کارکرد آن ملموس و مشخص نیست. عملکرد کلی شرکت مورد ابهام بوده و این موضوع ارتکاب کلاهبرداری توسط شرکت را محتمل می سازد. از کاربران و علاقه مندان به مشارکت در شرکت های جذب سرمایه توصیه می شود به منظور پیشگیری از وقوع هرگونه کلاهبرداری از فعالیت در این شرکت خودداری نموده و سرمایه خودشان را در معرض خطر قرار ندهند. 

**قوانینی که نقض شده است:**

**نوع هشدار:**
هشدار با ریسک بالا
**نوع کلاهبرداری:** 
پانزی
**وضعیت کلاهبرداری:**
در جریان
**نام کلاهبرداران:**

**نام شرکت(ها):**
اوربیت نتورک
**آدرس وب سایت(ها):**
https://www.orbitnetwork.net/
**برآورد مبلغ کلاهبرداری:**

**منابع:**

https://www.iranhoshdar.ir/Default.aspx?PageName=Newsb&ID=41156

**جمع بندی:**

**Comment by Blockunity** [comment]
تصحیح اسکم ارز sln اشتباه تاپی هست و نماد اصلی
SNL
![Screenshot_20220630-142911_Chrome.jpg](https://user-images.githubusercontent.com/72666334/176677224-48277ac0-ecdb-4736-92ac-7a343f6c33fa.jpg)

https://orbitnetwork.com/latest-projects

**Comment by micro73** [comment]
![1](https://user-images.githubusercontent.com/103103225/183823754-d6126037-007a-4035-a24f-4cd73078f630.jpg)



هویت صاحب وب سایت در WHOIS پنهان است

رتبه ترانکو (چقدر ترافیک) نسبتاً پایین است

سرویس‌های ارزهای دیجیتال شناسایی شده‌اند، اینها می‌توانند ریسک بالایی داشته باشند
به نظر می رسد خدمات مالی یا محتوایی با ریسک بالا ارائه می شود
خدمات مالی ریسک/بالا ارائه می شود

روش‌های پرداخت از تراکنش‌های ناشناس پشتیبانی می‌کنند


![2](https://user-images.githubusercontent.com/103103225/183823825-92768da4-3d0e-46cd-9b5a-3fc1b2665680.jpg)


و اینجا هم بیشتر ورودی های سایت ایرانی هستند 


![3](https://user-images.githubusercontent.com/103103225/183824062-b532befd-5608-4055-a2f6-710c86788daa.jpg)


تاریخ ساخته شدن سایت 


![4](https://user-images.githubusercontent.com/103103225/183824105-f82d5016-8134-44c7-9b44-4c97137d341d.jpg)


و همچنان هویت گردانندگان مخفی
